### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by anas599**
In this PR:

- Added routes as described (attached find a screenshot)
- Added An action in the correct controller.
- Added A view file.
Screenshot 2023-05-08 at 5 03 21 PM

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @anas599, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the PR lacks description as we can see in the screenshot below
Screenshot from 2023-05-08 14-04-56


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'user#index'
+  get 'users/:id', to: 'user#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/', to: 'user#index'
+  get 'users/:id/posts/:id', to: 'posts#show'
```
Comment (gemmen29) [inline config/routes.rb:7]: - [ ] Kindly write the routes this way:
```ruby
     resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
as it's more professional and easier to read.

**Review by gemmen29 — APPROVED** [review]
Hi @anas599, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @anas599, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**